templates/settings.yml.erb

```diff
@@ -71,12 +73,13 @@
 # https is enabled if certificate, CA certificate, and private key are present in locations specifed by
 # ssl_certificate, ssl_ca_file, and ssl_private_key correspondingly
 # default values for https_port is 8443
-<% if ssl = scope.lookupvar("foreman_proxy::ssl") -%>
-#:http_port: 8000
-:https_port: <%= scope.lookupvar("foreman_proxy::port") %>
-<% else -%>
-#:https_port: 8443
-:http_port: <%= scope.lookupvar("foreman_proxy::port") %>
+<% if ssl -%>
+<%= "#:http_port: 8080\n" if !http -%>
```

Comment:
That is to say I want it to render like this:

```
# https is enabled if certificate, CA certificate, and private key are present in locations specifed by
# ssl_certificate, ssl_ca_file, and ssl_private_key correspondingly
# default values for https_port is 8443
#:http_port: 8080
https_port: 8443
```

I think you're suggestion would render it like this:

```
# https is enabled if certificate, CA certificate, and private key are present in locations specifed by
# ssl_certificate, ssl_ca_file, and ssl_private_key correspondingly
# default values for https_port is 8443
https_port: 8443
#:http_port: 8080
```

Is that too nitpicky on my part?